src/components/TokenSafety/index.tsx

```diff
@@ -85,7 +93,7 @@ const Buttons = ({
   return warning.canProceed ? (
     <>
       <StyledButton onClick={onContinue}>
-        <Trans>I understand</Trans>
+        {warning.level === WARNING_LEVEL.MEDIUM ? <Trans>Continue</Trans> : <Trans>I understand</Trans>}
```

Comment:
should this be `!displayWarningCopy`?

Reply:
If by `displayWarningCopy` you mean `displayWarningLabel`, then yes